cowamm/profitability/competitor_kpis/curve/curve_largest_2token_pools_4232976.sql

```diff
@@ -0,0 +1,65 @@
+-- Finds all the curve pools with 2 tokens and their TVLs
+-- Input: blockchain
+
+with 
+-- filters pools with 2 tokens
+pools as(
+select pool_address as contract_address,
+coin0 as token0,
+coin1 as token1,
+mid_fee*power(10,-10) as fee
+from curvefi_{{blockchain}}.view_pools
+where coin2 = 0x0000000000000000000000000000000000000000),
```

Comment:
Why is this needed? Intuitively this looks to me as if this is an invalid pool (with coin2 being the 0 address). Could you add a comment in code please?

Reply:
Curve can have multi-tokens pools, which would not make sense to compare with CoW AMM for now